commands/alias.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-nested-ternary */
+const ora = require('ora');
+const { configStore, getDefaults } = require('@eik/common-config-loader');
+const { getCWD, logger } = require('../utils');
+
+const Alias = require('../classes/alias');
+const { Alias: AliasFormatter } = require('../formatters');
+
+exports.command = 'alias <name> <version> <alias>';
```

Comment:
I see that all files in `commands` uses `exports` but all other files in this project uses `module.exports`. Ideally we should use the later. Though; I'm not gonna require that we change this since we want to convert the cli to ESM asap.